app/controllers/suggestion.rb

```diff
@@ -0,0 +1,77 @@
+get '/suggestions' do
+  @comments = Comment.all
+  @suggestions = Suggestion.all #define instance variable for view
```

Comment:
No need to assign all comments to `@comment` here, since a comment has no value apart from the suggestion it is commenting on. (It seems like you realize this in the `suggestions/index` view, where you correctly iterate through suggestions, then through each suggestion's comments.)

Reply:
Also FYI you are not actually using `@comments` (which is a good thing).